apps/storybook/package.json

```diff
@@ -31,6 +31,7 @@
     "react-dom": "17.0.2",
     "react-icons": "4.3.1",
     "react-suspense-fetch": "0.4.0",
+    "react-three-fiber": "6.0.13",
```

Comment:
`@react-three/fiber` is already installed so I think this is the old package ?

Reply:
Hmm yep :scream: